src/mame/audio/phoenix.cpp

```diff
@@ -101,7 +101,6 @@ void phoenix_sound_device::device_start()
 	save_item(NAME(m_noise_state.polyoffs));
 	save_item(NAME(m_noise_state.lowpass_counter));
 	save_item(NAME(m_noise_state.lowpass_polybit));
-	save_pointer(NAME(m_poly18), (1ul << (18-5)));
```

Comment:
The reasoning for this being saved was probably so you can see/hack it in debugger memory views.  Not sure if anyone actually used it.